src/mds/MDSRank.h

```diff
@@ -433,7 +435,6 @@ class MDSRank {
     friend class C_MDS_MonCommand;
 
     const mds_rank_t whoami;
-    std::string fs_name;
```

Comment:
This moves it into the public section. The correct fix is to adjust the constructor init ordering.

Reply:
oops! I wrongly interpreted the original access of `fs_name` as public. Will fix it.